pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -925,7 +928,7 @@ func TestStaticPolicyStartWithResvList(t *testing.T) {
 			reserved:        cpuset.New(0, 1),
 			stAssignments:   state.ContainerCPUAssignments{},
 			stDefaultCPUSet: cpuset.New(2, 3, 4, 5),
-			expErr:          fmt.Errorf("not all reserved cpus: \"0-1\" are present in defaultCpuSet: \"2-5\""),
+			expErr:          errors.New("current set of available CPUs \"0-11\" doesn't match with CPUs in state \"0-5\""),
```

Comment:
This doesn't seem right. The reserved cpus are 0,1 so available/default should not include them and the error message. I think the error message should be something like below:
```suggestion
			expErr:          fmt.Errorf("current set of available CPUs \"2-11\" doesn't match with CPUs in state \"2-5\""),
```

Also, as per my one of previous suggestion, I think we should use continue using `fmt.Errorf` instead of `errors.New`.